mteb/tasks/STS/rus/RUParaPhraserSTS.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskSTS import AbsTaskSTS
+
+
+class RUParaPhraserSTS(AbsTaskSTS):
+    metadata = TaskMetadata(
+        name="RUParaPhraserSTS",
+        dataset={
+            "path": "merionum/ru_paraphraser",
+            "revision": "43265056790b8f7c59e0139acb4be0a8dad2c8f4",
+        },
+        description="Human-annotated STS dataset of Korean reviews, news, and spoken word sets. Part of the Korean Language Understanding Evaluation (KLUE).",
+        reference="https://arxiv.org/abs/2105.09680",
+        type="STS",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["rus-Cyrl"],
+        main_score="cosine_spearman",
+        date=("2009-01-01", "2019-01-01"),  # rough estimate,
+        form=["written"],
+        domains=["News"],
+        task_subtypes=[],
+        license="MIT",
+        socioeconomic_status="high",
```

Comment:
This part I am totally uncertain about but went with `"high"` as it seemed most appropriate.

Reply:
Since these are from news headlines, I would say `mixed` would be more appropriate.